### ISSUE TITLE
[Feature]: create-playwright

### ISSUE THREAD

**Issue by karlhorky** [microsoft/playwright#33243]
### 🚀 Feature Request

Add `--no-gha` option to [`create-playwright`](https://github.com/microsoft/create-playwright) to be able to skip the question of creating the GitHub Actions workflow, instead of requiring interactive input from the user

Already opened a PR for this, but it was closed because there's no companion issue yet:

- https://github.com/microsoft/create-playwright/pull/130

### Example

```bash
$ pnpm create playwright
Getting started with writing end-to-end tests with Playwright:
Initializing project in '.'
✔ Do you want to use TypeScript or JavaScript? · TypeScript
✔ Where to put your end-to-end tests? · tests
✔ Add a GitHub Actions workflow? (y/N) · false
✖ Install Playwright browsers (can be done manually via 'pnpm exec playwright install')? (Y/n) · true

$ pnpm create playwright --no-gha
Getting started with writing end-to-end tests with Playwright:
Initializing project in '.'
✔ Do you want to use TypeScript or JavaScript? · TypeScript
✔ Where to put your end-to-end tests? · tests
✖ Install Playwright browsers (can be done manually via 'pnpm exec playwright install')? (Y/n) · true
```

### Motivation

Be able to run the script in a more automated, zero-config way, instead of having to input details interactively as a user.

**Comment by tphillips-ict** [comment]
I am not really sure how removing one question from a list of 4 makes the initialisation more script-friendly.
Maybe the responses to the questions should be able to be given on the command-line when running `create playwright`

**Comment by karlhorky** [comment]
> Maybe the responses to the questions should be able to be given on the command-line when running `create playwright`

Yes, agreed. This is a common pattern with scaffolding CLIs like `create-next-app` - interactive questions should be specifiable via command line flags too.

> Does `--quiet` achieve the same effect?

No, the `--quiet` flag is an unconfigurable list of defaults. (exception being [the `TEST_OPTIONS` environment variable, which is discouraged (undocumented and internal)](https://github.com/microsoft/playwright/issues/29893#issuecomment-1991455649))

**Comment by karlhorky** [comment]
Superseded by #39987
